.github/workflows/add-codeql-prs-to-project.yml

```diff
@@ -0,0 +1,22 @@
+name: Add CodeQL pull requests to project
+
+on:
+  pull_request:
+    types:
+      - opened
+      - labeled
+
+jobs:
+  add-codeql-static-prs-to-project:
+    name: Add CodeQL static team PRs to project
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: read
+      repository-projects: write
+    steps:
+      - uses: actions/add-to-project@cb277e5c4351a2c0fdb00059bb8fdedb309a7957
```

Comment:
As this is a first-party action, you can pull it in via a tag or branch - it is only 3rd-party actions where we need to fix the commit SHA